wolox-books-vue/src/views/auth/screens/bookList/BookList.vue

```diff
@@ -4,22 +4,16 @@
 </template>
 
 <script>
-import { BooksService } from '../../../../services/booksService'
 import BookCard from './components/BookCard.vue'
 
-const booksService = new BooksService()
-
 export default {
-  data: () => {
-    return {
-      books: null
+  computed: {
+    books () {
```

Comment:
use [mapActions](https://vuex.vuejs.org/guide/actions.html#dispatching-actions-in-components) from vuex